Jetsnack/app/src/main/java/com/example/jetsnack/ui/JetsnackApp.kt

```diff
@@ -16,36 +16,19 @@
 
 package com.example.jetsnack.ui
 
-import androidx.activity.OnBackPressedDispatcher
-import androidx.compose.animation.Crossfade
 import androidx.compose.runtime.Composable
-import androidx.compose.runtime.remember
-import androidx.compose.runtime.saveable.rememberSaveable
-import com.example.jetsnack.ui.home.Home
-import com.example.jetsnack.ui.snackdetail.SnackDetail
+import androidx.navigation.compose.rememberNavController
 import com.example.jetsnack.ui.theme.JetsnackTheme
-import com.example.jetsnack.ui.utils.Navigator
 import com.google.accompanist.insets.ProvideWindowInsets
 
 @Composable
-fun JetsnackApp(backDispatcher: OnBackPressedDispatcher) {
-    val navigator: Navigator<Destination> = rememberSaveable(
```

Comment:
Can the `Navigator` class now be removed?

Reply:
Done!